src/pkg/clouds/gcp/account.go

```diff
@@ -7,17 +7,40 @@ import (
 	"errors"
 	"fmt"
 	"log"
+	"os"
+	"path/filepath"
 	"strings"
 
 	"golang.org/x/oauth2/google"
+	ini "gopkg.in/ini.v1"
 )
 
 var FindGoogleDefaultCredentials func(ctx context.Context, scopes ...string) (*google.Credentials, error) = google.FindDefaultCredentials
 
+func GetGcloudAccountEmail() (string, error) {
```

Comment:
This function does not rely on anything in this file so we should move it to its own file. How reusable is this function? Consider moving it to somewhere under the `cmd/cli` package if it's purely for the hint.